tests/test_ops.py

```
@@ -1,5 +1,6 @@
 import copy
 import pathlib
+import shutil
```

Ruff (0.8.2)

`shutil` imported but unused

Remove unused import: `shutil`

(F401)